AUTHORS

```diff
@@ -0,0 +1,166 @@
+theorygeek <theorygeek@gmail.com>
+charlieschwabacher <charlieschwabacher@gmail.com>
+Dave Qorashi <dave@zeemee.com>
+Evan Huus <eapache@gmail.com>
+mvgijssel <maarten@hackerone.com>
+Chris Hanks <christopher.m.hanks@gmail.com>
+Jérémie Astori <jeremie@astori.fr>
+Scott Walkinshaw <scott.walkinshaw@gmail.com>
+Gaurav Tiwari <gaurav@gauravtiwari.co.uk>
+kliu <kliu@tribpub.com>
+Samuel Gwilym <sam@comique.co>
+Brian Krausz <bkrausz@stripe.com>
+Charlie Somerville <charlie@charliesomerville.com>
+alex <alesssandro@ukr.net>
+Kristian Kvarnsbacke <kristian@kmja.com>
+Todd Berman <tberman@github.com>
+Stanisław Chmiela <sjchmiela@gmail.com>
+Willian van der Velde <mail@willian.io>
+Keith Pitt <me@keithpitt.com>
+Fatih Kadir Akın <fakin@protein.tech>
+jden <jden@zendesk.com>
+Juan Lulkin <maiz@lulk.in>
+Michael Crismali <michael.crismali@gmail.com>
+Kyle Daigle <kyle.daigle@github.com>
+Brandon Black <brandonmblack@gmail.com>
+Justin Li <jli.justinli@gmail.com>
+Seba Gamboa <me@sagmor.com>
+Matt Colyer <matt@colyer.name>
+Ryan Collins <RyanCCollins@users.noreply.github.com>
+Samuel Brandão <gb.samuel@gmail.com>
+Sun Yaozhu <yzyzsun@gmail.com>
+Tiago Freire <tcfonnet@gmail.com>
+Valery Bugakov <skymk1@gmail.com>
+Wenley Tong <wenley.tong@gmail.com>
+Wesley Ellis <tahnok@gmail.com>
+Zach Holman <zach@zachholman.com>
+Zach Kemp <zvkemp@gmail.com>
+Zoltan Olah <zol@percolatestudio.com>
+aviscasillas <aviscasillas@gmail.com>
+empunkt <mfa@gpredictive.de>
+exAspArk <exaspark@gmail.com>
+jordanielewski <jordanielewski@users.noreply.github.com>
+willian <willian@hackerone.com>
+Alberto Miedes Garcés <albertomg994@gmail.com>
+zjlovezj <zjlovezj@gmail.com>
+Aldrin Martoq <aldrin@martoq.cl>
+Andrew Bennett <potatosaladx@gmail.com>
+Bala Paranj <bparanj@gmail.com>
+Blake Gentry <bgentry@users.noreply.github.com>
+Brooks Swinnerton <bswinnerton@gmail.com>
+Daisuke Taniwaki <daisuketaniwaki@gmail.com>
+Daniel Rinehart <danielr@philo.com>
+David Celis <me@davidcel.is>
+David North <davidnorth@gmail.com>
+David Qorashi <davoud.qorashi@gmail.com>
+Dmitry Polushkin <dmitry.polushkin@gmail.com>
+Elad Ossadon <elado7@gmail.com>
+Faris Mustafa <farismmk@gmail.com>
+Gabriel Reis <gabriel.oreis@gmail.com>
+Gabriele Renzi <rff.rff@gmail.com>
+Gerald Monaco <gerald@gmonaco.me>
+Hibariya <hibariya@gmail.com>
+Holman Gao <holman@golmansax.com>
+Ivan Povalyukhin <ipovalyukhin@yammer-inc.com>
+James MacAulay <james@shopify.com>
+James Phillips <jamesdphillips@gmail.com>
+John Bohn <jjbohn@gmail.com>
+Jonathan Younger <jonathan@daikini.com>
+Keith <keithsalisbury@gmail.com>
+Matthijs Nieuwboer <m.nieuwboer@youngcapital.nl>
+Mick Staugaard <mick@staugaard.com>
+Netto Farah <nettofarah@gmail.com>
+Nick Poorman <nickpoorman@gmail.com>
+Oguz Bilgic <fisyonet@gmail.com>
+Pol Miro <polmiro@gmail.com>
+Richard Lee <dlackty@gmail.com>
+Robin Murphy <robin@robinmurphy.co.uk>
+Robert Mosolgo <rdmosolgo@gmail.com>
+Christian Joudrey <cmallette@gmail.com>
+Garen Torikian <gjtorikian@gmail.com>
```

Comment:
This list, while nice, is going to be impossible to keep updated.

I'm also really uncomfortable with the idea of a list that's just ripe for spammers to pluck from: "Hello I am working with an early stage dark horse pre-seed startup and we are using the GraphQLs to help revolutionize chicken farmers! Would you be open for a ten minute chat?"

Can we remove this?